Libs/MRML/DisplayableManager/vtkMRMLThreeDViewInteractorStyle.cxx

```diff
@@ -105,6 +105,17 @@ bool vtkMRMLThreeDViewInteractorStyle::DelegateInteractionEventToDisplayableMana
 
   ed->SetAttributesFromInteractor(this->GetInteractor());
 
+  if (inputEventData->GetType() == vtkCommand::MouseMoveEvent)
+  {
+    // Update the cursor position
+    vtkMRMLScene * scene = this->GetCameraNode()->GetScene();
+    vtkMRMLCrosshairNode * crosshairNode = vtkMRMLCrosshairDisplayableManager::FindCrosshairNode(scene);
+    if (crosshairNode)
+    {
+      crosshairNode->SetCursorPositionRAS(worldPosition);
```

Comment:
only set it if it is valid (`QuickPick` returned true)

Maybe you can move all the new lines inside the `if (this->QuickPick(...))` branch.